<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1881</width>
    <height>1033</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Belki</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/index.qrc">
    <normaloff>:/appicon</normaloff>:/appicon</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topBar" native="true">
      <layout class="QHBoxLayout" name="_2">
       <item>
        <widget class="QLabel" name="datasetLabel">
         <property name="text">
          <string>Dataset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="datasetSelect">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="structureLabel">
         <property name="text">
          <string>Structure:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="structureSelect">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="granularitySlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Adjust granularity in cluster hierarchy</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="famsKSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Adjust parameter k for adaptive bandwidths</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadDataset"/>
   <addaction name="actionSplice"/>
   <addaction name="actionShowStructure"/>
   <addaction name="actionPersistAnnotations"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Markers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="markerWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="protSearch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Search Protein Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="protList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="loadMarkersButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="saveMarkersButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="clearMarkersButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Proteins under cursor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="ProfileWidget" name="profiles"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1881</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionLoadDataset"/>
    <addaction name="actionLoadDatasetAbundance"/>
    <addaction name="actionSplice"/>
    <addaction name="separator"/>
    <addaction name="actionLoadDescriptions"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMarkers">
    <property name="title">
     <string>&amp;Markers</string>
    </property>
    <addaction name="actionLoadMarkers"/>
    <addaction name="actionSaveMarkers"/>
    <addaction name="separator"/>
    <addaction name="actionClearMarkers"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuStructure">
    <property name="title">
     <string>Structure</string>
    </property>
    <addaction name="actionImportStructure"/>
    <addaction name="actionExportAnnotations"/>
    <addaction name="separator"/>
    <addaction name="actionShowStructure"/>
    <addaction name="separator"/>
    <addaction name="actionPersistAnnotations"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionCloseAllTabs"/>
    <addaction name="separator"/>
    <addaction name="actionUseOpenGL"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStructure"/>
   <addaction name="menuMarkers"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoadDataset">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Dataset</string>
   </property>
   <property name="toolTip">
    <string>Load a .tsv file with profiles or scored pairs</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quick &amp;Help</string>
   </property>
  </action>
  <action name="actionLoadMarkers">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Markers</string>
   </property>
   <property name="toolTip">
    <string>Load a text file with list of marker protein names</string>
   </property>
  </action>
  <action name="actionSaveMarkers">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Markers</string>
   </property>
   <property name="toolTip">
    <string>Save a text file with list of marker protein names</string>
   </property>
  </action>
  <action name="actionClearMarkers">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Markers</string>
   </property>
   <property name="toolTip">
    <string>Remove all current markers</string>
   </property>
  </action>
  <action name="actionShowStructure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/icons-custom/index.qrc">
     <normaloff>:/icons/show-partition.svg</normaloff>:/icons/show-partition.svg</iconset>
   </property>
   <property name="text">
    <string>Show Structure</string>
   </property>
   <property name="toolTip">
    <string>Color samples by their annotations/cluster</string>
   </property>
  </action>
  <action name="actionImportStructure">
   <property name="icon">
    <iconset theme="document-import">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Import Structure</string>
   </property>
   <property name="toolTip">
    <string>Load a .tsv annotition or .json clustering file</string>
   </property>
  </action>
  <action name="actionProfileView">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="view-fullscreen">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open Full Plot</string>
   </property>
   <property name="toolTip">
    <string>Show proteins under cursor in new window</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionExportAnnotations">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Export Annotations</string>
   </property>
   <property name="toolTip">
    <string>Save current annotations to a .tsv file</string>
   </property>
  </action>
  <action name="actionLoadDescriptions">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Load Protein &amp;Descriptions</string>
   </property>
   <property name="toolTip">
    <string>Load a .tsv description file</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionSplice">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Splice dataset</string>
   </property>
   <property name="toolTip">
    <string>Create a new dataset from the features of the current dataset</string>
   </property>
  </action>
  <action name="actionLoadDatasetAbundance">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Load &amp;Abundance Values</string>
   </property>
   <property name="toolTip">
    <string>Load a .tsv file and read abundance values</string>
   </property>
  </action>
  <action name="actionPersistAnnotations">
   <property name="icon">
    <iconset theme="document-new-from-template">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Keep as Annotations</string>
   </property>
   <property name="toolTip">
    <string>Extract current result and keep under a new name</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="actionCloseAllTabs">
   <property name="icon">
    <iconset theme="tab-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Close all tabs</string>
   </property>
  </action>
  <action name="actionUseOpenGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="show-gpu-effects">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Use &amp;accelerated graphics</string>
   </property>
   <property name="toolTip">
    <string>Use OpenGL acceleration for charts. May be faster or slower.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProfileWidget</class>
   <extends>QWidget</extends>
   <header>profiles/profilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/index.qrc"/>
  <include location="../resources/icons-custom/index.qrc"/>
 </resources>
 <connections/>
</ui>
